addons/mail/static/tests/helpers/discuss/setup_manager_patch.js

```diff
@@ -1,14 +1,16 @@
 /* @odoo-module */
 
 import { discussTypingService } from "@mail/discuss/typing/typing_service";
+import { discussStoreService } from "@mail/discuss/discuss_store_service";
 import { setupManager } from "@mail/../tests/helpers/webclient_setup";
 import { patch } from "@web/core/utils/patch";
 
-patch(setupManager, "discuss/typing", {
+patch(setupManager, "discuss", {
```

Comment:
Don't move this file, make a duplicate instead. Typing has its own folder and should be self contained withing it.